modules/builds-configmaps-use-case-consuming-in-volumes.adoc

```diff
@@ -0,0 +1,85 @@
+// Module included in the following assemblies:
+//
+//* builds/builds-configmaps.adoc
+
+[id="builds-configmaps-use-case-consuming-in-volumes_{context}"]
+= Consuming ConfigMaps in volumes
+
+A ConfigMap can also be consumed in volumes. Returning again to the following example ConfigMap:
+
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: special-config
+  namespace: default
+data:
+  special.how: very
+  special.type: charm
+----
+
+.Procedure
+
+You have a couple different options for consuming this ConfigMap in a volume.
+
+* The most basic way to consume a ConfigMap in a volume is to populate the volume with files where the key is the file name and the content of the file is the value of the key:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: dapi-test-pod
+spec:
+  containers:
+    - name: test-container
+      image: gcr.io/google_containers/busybox
+      command: [ "/bin/sh", "cat", "/etc/config/special.how" ]
+      volumeMounts:
+      - name: config-volume
+        mountPath: /etc/config
+  volumes:
+    - name: config-volume
+      configMap:
+        name: special-config
+  restartPolicy: Never
+----
++
+When this pod is run, the output will be:
++
+----
+very
+----
+
```

Comment:
Tried to run the above but do not see the output in the logs as specified here.

Reply:
Yes, @kasturinarra  and I tried this, if using  command: [ "/bin/sh", "cat", "/etc/config/special.how" ], pod will failed; and if using  command: [ "/bin/sh", "-c", "cat", "/etc/config/special.how" ], pod completes, but no output will be very.

Reply:
Thank you, both! For this one, should the "When this pod is run, the output will be" lines be removed in this example, or is does the YAML content need to change?

Reply:
I can see output: very from container with the command: http://pastebin.test.redhat.com/902137, not sure why pods log have nothing.